README.md

```diff
@@ -90,9 +90,9 @@ No tests here
 
 👤 **Author**
 
-- GitHub: [@Ademola-coding](https://github.com/Ademola-coding)
-- Twitter: [@devAdemolaOfNig](https://twitter.com/Ademola-coding)
-- LinkedIn: [Ademola Owoeye](https://linkedin.com/in/ademola-coding)
+- GitHub: [Ademola-coding](https://github.com/Ademola-coding)
+- Twitter: [@devAdemolaOfNig](https://twitter.com/devAdemolaOfNig)
+- LinkedIn: [Ademola Owoeye](www.linkedin.com/in/ademola-owoeye)
```

Comment:
- [ ] Kindly fix the link to your LinkedIn page as requested by the  previous reviewer it is still broken 👍🏿 
<img width="934" alt="o" src="https://github.com/Ademola-coding/vet_clinic/assets/69638013/c1d053ce-52cd-41c4-9cd1-4be292689a56">

Reply:
you reviewed the dev branch, it has been fixed on this branch. and it will reflect after I merge it
@prolajumokeoni

Reply:
DONE